### PR TITLE
Add basic info about the app

### PR DESCRIPTION
**Pull request by evulins**
Make README beautiful again!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,85 @@
-# Getting Started
+# Thinking Classrooms UI
 
-Required Node version: 16.13.0
+**Thinking Classrooms** project is created to help math teachers in Poland to share theirs ideas for conducting lessons according to the Thinking Classroom. Through our platform, they will be able to easily search for an interesting issue along with ideas for math classes.
```
Comment (franiakat) [inline README.md]:
```suggestion
**Thinking Classrooms** project is created to help math teachers in Poland to share theirs ideas for conducting lessons according to the Thinking Classroom. method Through our platform, they will be able to easily search for an interesting issue along with ideas for math classes.
```

File: README.md [hunk 2]
```diff
@@ -1,70 +1,85 @@
...
 
-## Available Scripts
+**Thinking Classrooms** platofrm is a MVP version which will be developed in accordance with the needs of its users.
```
Comment (franiakat) [inline README.md]:
```suggestion
**Thinking Classrooms** platform is an MVP version which will be developed in accordance with the needs of its users.
```

File: README.md [hunk 3]
```diff
@@ -1,70 +1,85 @@
...
+- [How to run Thinking Classrooms in production mode](##how-to-run-thinking-classrooms-in-production-mode)
+- [Available Scripts](#available-scripts)
+- [License](#license)
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## About Thinking Classrooms method
 
-### `yarn test`
+**Thinking Classrooms** is the concept developed by Peter Liljedahl. The goal of Thinking Classrooms is to get students to solve problems by thinking through them until they arrive at a solution. Some of the takeaways from Liljedahl’s work are easy to implement.:
```
Comment (franiakat) [inline README.md]:
```suggestion
**Thinking Classrooms** is the concept developed by Peter Liljedahl. The goal of Thinking Classrooms is to get students to solve problems by thinking them through until they arrive at a solution. Some of the takeaways from Liljedahl’s work are easy to implement.:
```

File: README.md [hunk 4]
```diff
@@ -1,70 +1,85 @@
...
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- _Stand up._ Standing up while solving problems helps students remain engaged with the work before them. Standing also leads to collaboration.
```
Comment (franiakat) [inline README.md]:
```suggestion
- _Stand up._ Standing up while solving problems helps students remain engaged with the work in front of them. Standing also leads to collaboration.
```

File: README.md [hunk 5]
```diff
@@ -1,70 +1,85 @@
...
+- _Hint, don’t answer._ Give enough hints to keep your students on the right path, but don’t walk the path for them.
```
Comment (franiakat) [inline README.md:23]: <3

File: README.md [hunk 6]
```diff
@@ -1,70 +1,85 @@
...
+[https://zealous-spence-867386.netlify.app/](https://zealous-spence-867386.netlify.app/)
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Available Scripts
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+In the project directory, you can run:
 
-### Code Splitting
+### `yarn start`
```
Comment (franiakat) [inline README.md]:
```suggestion
# `yarn start`
```
or maybe

```suggestion
## `yarn start`
```

Comment (franiakat) [reply README.md]: Can be applied for other commands as well.

**Review by franiakat — COMMENTED** [review]
Beautiful <3 Can be viewed in preview here:
https://github.com/Nontrivial-Things/thinking_classrooms_client/tree/update-readme#thinking-classrooms-ui

File: README.md (hunk 2 above)
Comment (franiakat) [reply README.md]: the last bit ;)

**Review by bazelajoanna — APPROVED** [review]
Great! 👏

**Review by franiakat — APPROVED** [review]
sqash

**Pull request merged**